SS_write_report.tpl

```diff
@@ -3298,6 +3326,8 @@ FUNCTION void write_bigoutput()
   SS2out<<" end selex output "<<endl;
   }  // end do report detail
   wrote_bigreport++;
+  SS2out.close();
```

Comment:
@Rick-Methot-NOAA For some reason this line leads to Dynamic Bzero NOT being written in Report.sso. At the top of the report file it says `Y Dynamic_Bzero report:59` but the section is not written.

Reply:
it should be in report.sso.
I intentionally turned off the profiling outputs for the MCEVAL outputs but did not intentionally change dynamic Bzero.  I will chenck on this also